src/main/java/uk/gov/companieshouse/insolvency/delta/processor/InsolvencyDeltaProcessor.java

```diff
@@ -41,13 +52,18 @@ public InsolvencyDeltaProcessor(InsolvencyDeltaProducer deltaProducer,
     public void processDelta(Message<ChsDelta> chsDelta) {
         try {
             MessageHeaders headers = chsDelta.getHeaders();
+            final String logContext = chsDelta.getPayload().getContextId();
```

Comment:
We already retrieve the payload on line 59, could use it instead and this moves after and becomes `payload.getContextId()`

Reply:
Addressed.